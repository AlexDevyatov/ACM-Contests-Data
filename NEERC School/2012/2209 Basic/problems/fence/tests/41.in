15
-318 -60
-308 -197
-270 -274
-134 -301
-81 -310
205 -307
256 -154
305 58
299 175
278 213
230 292
168 313
-111 316
-265 298
-314 186
85 57
