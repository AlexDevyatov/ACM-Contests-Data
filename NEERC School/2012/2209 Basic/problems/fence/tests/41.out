16
-318 -60
-308 -197
-270 -274
-134 -301
-81 -310
205 -307
256 -154
305 58
299 175
85 57
278 213
230 292
168 313
-111 316
-265 298
-314 186
